toolkit/docs/CVE/CVE-Quickstart-Guide.md

```diff
@@ -0,0 +1,167 @@
+# **Document: Fixing CVE Patches for Azure Linux**
+
+This document provides a step-by-step guide to fixing CVE patches for Azure Linux. It includes identifying the CVE, locating and preparing the package, applying the patch, and submitting it for integration into Azure Linux.
+
```

Comment:
Shall I add the CVE Guide in ADO: https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/233/CVEs?